docs/Developer Documentation/ResumeController/index.md

```diff
@@ -81,7 +81,7 @@ Main public function for resumptions is `ResumptionDataProcessor::Restore` :
 ```cpp
   /**
   * @brief Running resumption data process from saved_app to application.
-  * @param application application which will be resumed
+  * @param application which will be resumed
   * @param saved_app application specific section from backup file
```

Comment:
```suggestion
  * @param application Application which will be resumed
  * @param saved_app Application specific section from backup file
  * @param callback Function signature to be called when
  * data resumption will be finished
```
This is actually intentional (though the capitalization might make it clearer). This is split into two parts `@param application` defines the name of the parameter, while the second part is the description for the parameter - "application which will be resumed"